include/phpFunctions.php

```diff
@@ -566,6 +785,39 @@ function GetRecentPost(){
 		return $result['recentpost'];
 	}
 
+	function EditPost($postID, $changes, $tagarray){
+		// echo"EditPost Called!";
+		var_dump($changes);
+		foreach($changes as $column=>$change){
```

Comment:
There are a couple issues with how you're doing this:

1) while you might not notice in your website since it's not getting any real traffic, you generally want to minimize the number of db queries you do because those can be the slowest part of loading a page (there's normally latency between the PHP server and the database server, but there's not in WAMP since they're on the same machine). So you're doing a bunch of updates here when it'd be better to do a single one where you update all the fields at once.

2) since you're passing `$column` into the query, that's a potential vector for SQL injection. You might happen to know for sure that it won't be an issue (like, if it's not based on user generated content) but the normal rule is that each function should ensure security without needing to trust whatever code is calling it. So if someone managed to get malicious code into `$changes`, this function would be a problem.